### PR TITLE
[BUG, MRG] Remove check on `mne.viz.Brain.add_volume_labels`

### PR DESCRIPTION
**Pull request by alexrockhill**
Freesurfer now has a nice `wmparc.mgz` that doesn't fit the `aseg` naming conventions. I think maybe just removing this check is easiest.

### CODE REVIEW

File: mne/viz/_brain/_brain.py [hunk 1]
```diff
@@ -2628,8 +2628,6 @@ def add_volume_labels(
         import nibabel as nib
 
         # load anatomical segmentation image
-        if not aseg.endswith("aseg"):
```
[the remark this answers is not in the document]
Comment (alexrockhill) [reply mne/viz/_brain/_brain.py:2631]: We don't check this kind of stuff elsewhere i.e. if you try and read a `trans` file, it doesn't check that it ends with `trans.fif` so maybe better just to let any readable file

[the remark this answers is not in the document]
Comment (larsoner) [reply mne/viz/_brain/_brain.py:2631]: > We don't check this kind of stuff elsewhere

Actually we do -- we have a filename checking function to do this sort of thing for raw/epochs/evoked. Given our tight coupling with FreeSurfer and how infrequently they add new stuff, I don't mind having to change the check every few years to accommodate a new name. In the meantime if we get rid of the check we risk users accidentally passing files that don't work and getting some cryptic error later. There's no telling how many times the current check has saved a user some pain but I bet it's a number greater than zero somewhere...

[the remark this answers is not in the document]
Comment (alexrockhill) [reply mne/viz/_brain/_brain.py:2631]: Sounds good, okay to change to a warning though?

[the remark this answers is not in the document]
Comment (larsoner) [reply mne/viz/_brain/_brain.py:2631]: Lets default to error and allow overriding with _om_missing style handling

[the remark this answers is not in the document]
Comment (alexrockhill) [reply mne/viz/_brain/_brain.py:2631]: I just used endswith `parc` or `aseg` to be slightly more permissive without adding an extra argument which I'm not sure is worth the added complexity

**Comment by larsoner** [comment]
Thanks @alexrockhill !

**Pull request merged**